### ISSUE TITLE
how to config --enable-preview when use javafx:jlink

### ISSUE THREAD

**Issue by GG-A** [openjfx/javafx-maven-plugin#77]
how to config `--enable-preview` in `javafx-maven-plugin` when use `javafx:jlink` command?

**Comment by betanzos** [comment]
Using [`options`](https://github.com/openjfx/javafx-maven-plugin#javafxrun-options).

```xml
<configuration>
    <options>
        <option>--enable-preview</option>
    </options>
</configuration>
```

**Comment by GG-A** [comment]
@betanzos this is `javafx:run` **options**, not `javafx:jlink` option, it's not working for `javafx:jlink`.

**Comment by betanzos** [comment]
No, `options` will are also included in the launcher that creates `javafx:jlink`

![Untitled](https://user-images.githubusercontent.com/4399580/79029922-3420e480-7b5c-11ea-837f-88467d0a457e.png)

**Comment by GG-A** [comment]
@betanzos  it's not working on windows, my configuration is:  
```
<plugin>
                <groupId>org.openjfx</groupId>
                <artifactId>javafx-maven-plugin</artifactId>
                <version>0.0.4</version>
                <configuration>
                    <source>14</source>
                    <target>14</target>
                    <release>14</release>
                    <mainClass>OpenJFXAndOpenJDK/com.ctxx.JFXMain</mainClass>
                    <options>
                        <option>--enable-preview</option> 
                    </options>

                    <stripDebug>false</stripDebug> 
                    <noHeaderFiles>true</noHeaderFiles>
                    <noManPages>true</noManPages>
                    <compress>2</compress>  
             
                    <jlinkImageName>openjfxApp</jlinkImageName>
                    <launcher>launcher</launcher> 
                </configuration>
            </plugin>
```
![1](https://user-images.githubusercontent.com/49749177/79030554-13907300-7bcc-11ea-97f9-919a7588be6b.png)

**Comment by betanzos** [comment]
So, this is a command bug in Windows since it should work like in Linux.

**Comment by abhinayagarwal** [comment]
Fixed by #79
